index.html

```diff
@@ -74,8 +71,308 @@ <h1>I'm Joel <br>Glad to see<br> you!</h1>
+                            <button type="button" class="p-text">
+                                See Project
+                            </button>
+                        </div>
+                    </div>
+                </div>
+        </section>
+      
+       <!-- About myself  -->
+       <div class="wrapper">
+    
+            <section class="about-me">
+                <h1>About<br>Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+    
+                    <span class="paragraph">LET'S CONNECT</span>
+                    <div class="social_icons">
+                        <ul class="font_icons">
+                            <li>
+                                <a href="#">
+                                    <img alt="twitter" src="images/twitter-icon.png"/>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#">
+                                    <img alt="link" src="images/Linkedin-icon.png"/>
+                                </a>
+                            </li>
+                            
+                            <li>
+                                <a href="#">
+                                    <img alt="v2" src="images/Vector-2.png"/>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#">
+                                    <img alt="git" src="images/Vector-github.png"/>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#">
+                                    <img alt="vector" src="images/Vector-finger.png"/>
+                                </a>
+                            </li>
+                        
+                        </ul>
+                        <div class="see-project get-resume">
+                            <button type="button" class="p-text">
+                                Get my resume
+                            </button>
+                        </div>
+                    </div>
+                    <div class="abt-self">
+                        <div class="arrow1">
+                            <p>Languages</p>
+                            <img alt="arr1" src="images/arrow-img.png">
+                        </div>
+                        <div class="code-icons">
+                          <ul class="code-list">
+                            <li>
+                              <img alt="js-icon" src="images/js-img.png">
+                              <p>Javascript</p>
+                            </li>
```

Comment:
- [ ] The technologies categories are a list and it is better to have them in a `<ul>` instead of a meaningless `<div>` tag. Also each individual technology should be a `<li>`. Please implement this change 🙏